pkg/graph/schema.graphql

```diff
@@ -665,6 +666,51 @@ type SystemIntake {
   updatedAt: Time!
 }
 
+"""
+Indicates which action should be taken
+"""
+enum SystemIntakeActionType {
+  BIZ_CASE_NEEDS_CHANGES
+  CREATE_BIZ_CASE
+  GUIDE_RECEIVED_CLOSE
+  ISSUE_LCID
+  NEED_BIZ_CASE
+  NO_GOVERNANCE_NEEDED
+  NOT_IT_REQUEST
+  NOT_RESPONDING_CLOSE
+  PROVIDE_FEEDBACK_NEED_BIZ_CASE
+  PROVIDE_GRT_FEEDBACK_BIZ_CASE_DRAFT
+  PROVIDE_GRT_FEEDBACK_BIZ_CASE_FINAL
+  READY_FOR_GRB
+  READY_FOR_GRT
+  REJECT
+  SEND_EMAIL
+  SUBMIT_BIZ_CASE
+  SUBMIT_FINAL_BIZ_CASE
+  SUBMIT_INTAKE
+}
+
+"""
+An action taken on a system intake, often resulting in a change in status.
+"""
+type SystemIntakeAction {
+  id: UUID!
+  systemIntake: SystemIntake!
+  type: SystemIntakeActionType!
+  actor: SystemIntakeActionActor!
+  feedback: String
+  createdAt: Time!
+}
+
+"""
+A person performing an action on a system intake
+"""
+type SystemIntakeActionActor {
+  name: String!
+  email: String!
+  eua: String!
```

Comment:
I don't think we actually use this value in the front end. It's included in the REST API and so I kept it, but we might want to take out anything that isn't really being used.

Reply:
I concur, since it seems to me the schema is mostly intended to represent front-end needs/wants, if a property isn't used I'd keep it out of the schema until it is needed.